src/TextUI/ResultPrinter.php

```diff
@@ -422,6 +452,20 @@ public function addFailure(PHPUnit_Framework_Test $test, PHPUnit_Framework_Asser
     }
 
     /**
+     * A failure occurred.
```

Comment:
The same of my [last comment](https://github.com/sebastianbergmann/phpunit/pull/1838/files#r38142011).